README.md

```diff
@@ -14,58 +14,75 @@
+| [AXA Footer Small](./src/components/20-molecules/footer-small)                     | [![npm version](https://img.shields.io/npm/v/@axa-ch/footer-small.svg?style=flat)](https://www.npmjs.com/package/@axa-ch/footer-small)                     |
+| [AXA Icon](./src/components/10-atoms/icon)                                         | [![npm version](https://img.shields.io/npm/v/@axa-ch/icon.svg?style=flat)](https://www.npmjs.com/package/@axa-ch/icon)                                     |
+| [AXA Input File](./src/components/10-atoms/input-file)                             | [![npm version](https://img.shields.io/npm/v/@axa-ch/input-file.svg?style=flat)](https://www.npmjs.com/package/@axa-ch/input-file)                         |
+| [AXA Input Text](./src/components/10-atoms/input-text)                             | [![npm version](https://img.shields.io/npm/v/@axa-ch/input-text.svg?style=flat)](https://www.npmjs.com/package/@axa-ch/input-text)                         |
+| [AXA Link](./src/components/10-atoms/link)                                         | [![npm version](https://img.shields.io/npm/v/@axa-ch/link.svg?style=flat)](https://www.npmjs.com/package/@axa-ch/link)                                     |
+| [AXA Materials](./src/components/00-materials)                                     | [![npm version](https://img.shields.io/npm/v/@axa-ch/materials.svg?style=flat)](https://www.npmjs.com/package/@axa-ch/materials)                           |
+| [AXA Table](./src/components/30-organisms/table)                                   | [![npm version](https://img.shields.io/npm/v/@axa-ch/table.svg?style=flat)](https://www.npmjs.com/package/@axa-ch/table)                                   |
+| [AXA Table Sortable](./src/components/30-organisms/table-sortable)                 | [![npm version](https://img.shields.io/npm/v/@axa-ch/table-sortable.svg?style=flat)](https://www.npmjs.com/package/@axa-ch/table-sortable)                 |
+| [AXA Text](./src/components/10-atoms/text)                                         | [![npm version](https://img.shields.io/npm/v/@axa-ch/text.svg?style=flat)](https://www.npmjs.com/package/@axa-ch/text)                                     |
+| [AXA Textarea](./src/components/10-atoms/textarea)                                 | [![npm version](https://img.shields.io/npm/v/@axa-ch/textarea.svg?style=flat)](https://www.npmjs.com/package/@axa-ch/textarea)                             |
+| [AXA Top Content Bar](./src/components/20-molecules/top-content-bar)               | [![npm version](https://img.shields.io/npm/v/@axa-ch/top-content-bar.svg?style=flat)](https://www.npmjs.com/package/@axa-ch/top-content-bar)               |
 
 ## Alpha-Released Components
-- 
+
+- [AXA Popup](./src/components/20-molecules/popup) [![npm version](https://img.shields.io/npm/v/@axa-ch/popup.svg?style=flat)](https://www.npmjs.com/package/@axa-ch/popup)
```

Comment:
In my opinion you could use here also a table like you used it for the released components. Keeping in mind, that there could be more than one alpha-released component.

Reply:
Good input!

Reply:
I thought about that too, but imho we should never have too much alpha components that are not finished and production ready, otherwise we are having issues in our process. That would be an extra table that would mostly be empty, which also looks weird. I would keep the table for really finished and released stuff and the other sections more as "informative".

Reply:
Oke I'm happy with that 👍